IN-CORE/incore-docs
# 107 modify social vulnerability to query census data from censusutil instead of stored datasets

To test this notebook
- install this branch https://github.com/IN-CORE/pyincore/pull/156 by 
1. comment out the `pkg_resources.extern.packaging.version.Version = pkg_resources.SetuptoolsLegacyVersion` in setup.py
2. run `pip install . -U --no-dependencies`

- run this notebook

Note that the `National vulnerability feature averages` I calculated is different than the original file. You can compare these two on incore-dev
original: 6241d9c653302c512d67ef26
new: 62b4d24bbe53de4a4737df2f
